symengine/sets.h

```diff
@@ -0,0 +1,62 @@
+/**
+ *  \file sets.h
+ *  Polynomial Manipulation
+ *
+ **/
+#ifndef SYMENGINE_SETS_H
+#define SYMENGINE_SETS_H
+
+#include <symengine/basic.h>
+#include <symengine/functions.h>
+#include <symengine/number.h>
+#include <symengine/rational.h>
+#include <symengine/complex.h>
+
+
+namespace SymEngine {
+
+class Interval : public Basic  {
```

Comment:
Hmm, Sage uses the notation I mentioned. http://doc.sagemath.org/html/en/reference/rings_numerical/sage/rings/real_mpfi.html

It depends on what you are trying to represent here.
What are you trying to represent using an Interval